pkg/dashboards/kubernetes/apiserver/apiserver.go

```diff
@@ -0,0 +1,122 @@
+package apiserver
+
+import (
+	"github.com/perses/community-dashboards/pkg/dashboards"
+	panelsGostats "github.com/perses/community-dashboards/pkg/panels/gostats"
+	panels "github.com/perses/community-dashboards/pkg/panels/kubernetes"
+	"github.com/perses/community-dashboards/pkg/promql"
+	"github.com/perses/perses/go-sdk/dashboard"
+	panelgroup "github.com/perses/perses/go-sdk/panel-group"
+	listVar "github.com/perses/perses/go-sdk/variable/list-variable"
+	labelValuesVar "github.com/perses/plugins/prometheus/sdk/go/variable/label-values"
+)
+
+func withMarkdown(datasource string, labelMatcher promql.LabelMatcher) dashboard.Option {
+	return dashboard.AddPanelGroup("Notice",
+		panelgroup.PanelsPerLine(1),
```

Comment:
We can probably make this panel smaller
```suggestion
		panelgroup.PanelsPerLine(1),
		panelgroup.PanelHeight(3)
```

Reply:
done thanks